modules/installation-load-balancing-user-infra.adoc

```diff
@@ -142,6 +142,12 @@ Configure the following ports on both the front and back of the load balancers:
 |X
 |HTTP traffic
 
+|`1936`
+|The worker nodes that run the Ingress Controller pods, by default. You must configure the `/healthz/ready` endpoint for the API server health check probe.
```

Comment:
It is not for the `... API server health check probe` but `ingress health check probe`.

Reply:
acked